src/domain/__tests__/PreLoadRepository.test.ts

```diff
@@ -0,0 +1,20 @@
+import * as data from './data';
+import { contentTypeIsPdf } from '../PreLoadRepository';
+
+describe('PreLoadRepository', () => {
```

Comment:
Add missing test about `uniquePreloadIdx`

Reply:
You're right! I'm going to add the tests later!

Reply:
Done!